src/DynamoCore/Scheduler/DynamoScheduler.cs

```diff
@@ -80,6 +83,9 @@ public enum TaskProcessMode
         Asynchronous
     }
 
+    /// <summary>
+    /// DynamoScheduler is run in the second thread, with Scheduler Dynamo runs tasks asynchronously.
```

Comment:
This class represents Dynamo scheduler. All the tasks are scheduled on the scheduler. Also, these tasks runs async.